packages/breadcrumbs/src/styled/StyledBreadcrumb.ts

```diff
@@ -6,19 +6,24 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
-import { retrieveComponentStyles, isRtl } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.list';
 
-export const StyledBreadcrumb = styled.ol.attrs(props => ({
-  'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb'], {
-    // RTL
-    [BreadcrumbStyles['is-rtl']]: isRtl(props)
-  })
-}))`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+/**
+ * 1. <ol> reset.
+ */
+export const StyledBreadcrumb = styled.ol.attrs(() => {
+  return {
+    'data-garden-id': COMPONENT_ID,
+    'data-garden-version': PACKAGE_VERSION
+  };
+})`
+  ${props => `
+    display: flex; ${/* [1] */ ''}
+    margin: 0; ${/* [1] */ ''}
+    padding: 0; ${/* [1] */ ''}
+    list-style: none; ${/* [1] */ ''}
+    font-size: ${props.theme.fontSizes.md};
+    direction: ${props.theme.rtl ? 'rtl' : 'ltr'};
```

Comment:
Usually we try to leave CSS defaults out of the render. This can just be `direction: ${props.theme.rtl && 'rtl'};` – styled-components will take care of the rest.

Reply:
Thanks for the suggestion 👍. I updated it so that CSS default is left out of the render.